test/library/standard/Math/victor-ludorum/testcases_gcd.chpl

```diff
@@ -0,0 +1,17 @@
+writeln(gcd(4,8));
```

Comment:
Can you put this test in  test/library/standard/Math ? Adding user-names in the test directory is a pattern we are trying to move away from.

Reply:
Okay sir !! I will do that . Actually previously this idea is preferred . Therefore, I have followed this